generator/ServiceClientGeneratorLib/Member.cs

```diff
@@ -972,10 +992,11 @@ public string DeprecationMessage
             {
                 string message = this.model.Customizations.GetPropertyModifier(this.OwningShape.Name, this._name)?.DeprecationMessage ??
                                  this.data[DeprecatedMessageKey].CastToString();
+                // TODO: Fill in s3 deprecation messages
```

Comment:
You resolved Norm's comment without a response (https://github.com/aws/aws-sdk-net/pull/3850#discussion_r2127588600) but did we decide not to fill in the deprecation messages now?